Assets/Scripts/Enemies/RangedEnemy.cs

```diff
@@ -33,6 +33,18 @@ public override void Attack()
         {
             Instantiate(projectile, launchPoint.transform.position, Quaternion.identity);
             timer = cooldown;
+            AudioManager.instance.CheckForLastPlayed(AudioManager.instance.soundEffects[3]);
         }
     }
+    public override void TakeDamage(int damage)
+    {
+        base.TakeDamage(damage);
+        AudioManager.instance.CheckForLastPlayed(AudioManager.instance.soundEffects[4]);
```

Comment:
Ideally you should have an enum for the Sound effects so 4, 5, 18 etc are possible to understand.